src/bot/logic/awareness.ts

```diff
@@ -215,14 +198,61 @@ export class MatchAwarenessImpl implements MatchAwareness {
                 .getPlayers()
                 .filter((p) => p !== playerData.name && !game.areAlliedPlayers(playerData.name, p));
             const enemy = game.getPlayerData(enemyPlayers[0]);
-            this.mainRallyPoint = getPointTowardsOtherPoint(
-                game,
-                playerData.startLocation,
-                enemy.startLocation,
-                10,
-                10,
-                0,
-            );
+            // 使用findPath找到从我方startLocation到enemy.startLocation的Track路径70%位置
+            const startTile = game.mapApi.getTile(playerData.startLocation.x, playerData.startLocation.y);
+            const targetTile = game.mapApi.getTile(enemy.startLocation.x, enemy.startLocation.y);
+            
+            if (startTile && targetTile) {
+                try {
+                    const path = game.mapApi.findPath(
+                        SpeedType.Track,
+                        false,
+                        { tile: startTile, onBridge: false },
+                        { tile: targetTile, onBridge: false }
+                    );
+                    
+                    if (path && path.length > 0) {
+                        // 选择路径70%位置的点
+                        const midPointIndex = Math.floor(path.length / 10 * 3);
+                        const midTile = path[midPointIndex].tile;
+                        this.mainRallyPoint = new Vector2(midTile.rx, midTile.ry);
+                        this.logger(`Rally point set to path 70% position: (${midTile.rx}, ${midTile.ry}), path length: ${path.length}`);
+                    } else {
+                        // 如果找不到路径，使用fallback方案
+                        this.mainRallyPoint = getPointTowardsOtherPoint(
+                            game,
+                            playerData.startLocation,
+                            enemy.startLocation,
+                            10,
+                            10,
+                            0,
+                        );
```

Comment:
```suggestion
                        this.mainRallyPoint = fallbackRallyPoint;
```

and so on below (instead of repeating getPointTowardsOtherPoint 3 times for 3 fallbacks)